guides/developers-guide/concepts/products-variants-options.md

```diff
@@ -0,0 +1,121 @@
+
+- **Attribute label** - Required. The attribute label describes the category of variant, for example, "Color" or "Size". In most cases this will be the same for all variants at the same level.
+- **Short title** - Required. The short title is the value for the attribute label. For example, a variant with attribute label "Color" might have short title "Red". This is usually shown in a select list or button set on a product detail page in your storefront.
+- **Title** - Required. The full title is usually shown on cart, checkout, and order summaries and on invoices. It should fully describe the configured variant. For example, if this is an option with short title "Large", its parent variant has short title "Red", and the product title is "Fancy T-Shirt", then a good title might be "Fancy T-Shirt - Red - Large".
+- **Origin country** - Optional. A drop-down menu of countries.
+- **Price** - Required. The price that you are actually selling your product.
+- **Compare at price** - Optional. The suggested retail price of your product.
+- **Width, Length, Height, Weight** - Optional. The size and weight of your product
+- **Taxable** - Optional. Check this box to automatically add tax to this item when purchased. Add Tax Code and Tax Description for more options.
+- **Inventory tracking** - Optional. Check this box if you'd like to track this item in your inventory.
+- **Warn at** - Optional. Allows for `Limited Supply` notifications to be displayed on when quantity is lower than this number.
+- **Allow Backorder** - Optional. Allows customers to backorder the product.
+
+3. Make your variant visible by clicking "Make Visible" in the dropdown next to their name in the left sidebar.
+
+4. Click the **Publish** button on the top right to publish your product and all of its variants to your catalog.
+
+## Options
+
+If necessary, a variant can have further variants, which we call **“options”** or **“second level variants”**. For example, the “Size 9” variant for “Pair of Best Brand Shoes, Model B”, may come in two different colors, black and red. So the “Size 9” variant would have two variants of its own, “Black” and “Red”.
+
+It’s important to note that only a terminal variant may be added to a cart for checkout. So if a variant has options, a shopper will need to select one of the options, but if a variant has no options, that variant itself may be added to a cart to place an order.
+
+## Adding more product Variant Options
+
+1. Click on the button to the right of the variant you'd like to create an option for. Then, click "Add variant."
+
+2. Add the option's **Attribute Lanel**, **Label**, **Short Label**, **Quantity** and **Price**. **Label** is displayed in cart, checkout and orders. **Short Label** is displayed on the Product Detail Page. 
+
+3. Make the option visible from the left sidear, then click the **Publish** button on the top right to make your changes show up on the storefront.
+
+## Adding product images
```

Comment:
I am wondering if this duplicates content from the Shop Managers guide too much and if we should just update that and link to it?